src/Sulu/Bundle/AdminBundle/Resources/js/components/ColumnList/ToolbarButton.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import Icon from '../Icon';
+import type {ToolbarButton as ToolbarButtonProps} from './types';
+import toolbarStyles from './toolbar.scss';
+
+export default class ToolbarButton extends React.Component<ToolbarButtonProps> {
+    static defaultProps = {
+        skin: 'primary',
+    };
+
+    handleClick = () => {
+        this.props.onClick(this.props.index);
```

Comment:
Is this index in `this.props` the index of the column? This was now a little bit confusing for me... Shouldn't we name it `column` instead?